app/models/user.rb

```diff
@@ -44,8 +44,9 @@ class User < ApplicationRecord
 
   attr_accessor :remove_picture
 
-  validates :name, presence: true, format: { without: /\A["!@#$%^&]*\z/,
-                                             message: "can only contain letters and spaces" }
+  USERNAME_MAX_LENGTH = 50
+  validates :name, presence: true, length: { maximum: USERNAME_MAX_LENGTH }, format: { without: /\A["!@#$%^&]*\z/,
+                                                                                       message: "can only contain letters and spaces" }
```

Comment:
Let's also have a truncation logic for existing long name users to ensure the consistency.

> Better approach would be migration but risky as it includes the conflicts with user.